Automattic/wp-calypso
# Social Plans: Remove the advanced plan from the Social products

#### Proposed Changes

With the advanced plan in the social products array, we get a blank product appearing on the cloud.jetpack.com/pricing page. This PR removes it so that we can figure out what's going on!

#### Testing Instructions

* Go to jetpack.cloud.local:3000/pricing
* Without this change you'll see a blank social product in the products table
* With this change, the product will be removed
* Check that you can still purchase the advanced plan as with #69417

#### Pre-merge Checklist





Related to #69417
